src/CoreWCF.Primitives/src/CoreWCF/Channels/Message.cs

```diff
@@ -1196,7 +1196,7 @@ protected override void OnBodyToString(XmlDictionaryWriter writer)
         protected internal BodyWriter BodyWriter { get; private set; }
     }
 
-    internal abstract class ReceivedMessage : Message
+    public abstract class ReceivedMessage : Message
     {
```

Comment:
Don't make these implementations of Message public. Duplicate whatever is needed in WebHttp

Reply:
To be clear I should duplicate BufferedMessage and ReceivedMessage in WebHttp?

Reply:
Correct.

Reply:
I was able to make BufferedMessage and ReceivedMessage internal again, but a lot of other things had to have their accessibility modifiers changed. The changes were in 5e01034482906508b331e3e0fea40519eda4eecf. I'll leave this one unresovled in case this wasn't what you wanted.

Reply:
This is resolved.